speech/tests/unit/test__gax.py

```diff
@@ -16,6 +16,9 @@
 
 import mock
 
+from google.cloud import _helpers
+from google.cloud.gapic.speech.v1.speech_client import SpeechClient
```

Comment:
Danny and I have discussed offline and decided on "he who codes, rules". He cares more than I do so he is putting in an amendment to make the changes he wants, and then we will merge.